substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2213,7 +2192,7 @@ pub mod pallet {
 			// For now we only allow a pool to withdraw unbonded if its not destroying. If the pool
 			// is destroying then `withdraw_unbonded` can be used.
 			ensure!(pool.state != PoolState::Destroying, Error::<T>::NotDestroying);
-			pool.withdraw_from_staking(num_slashing_spans)?;
+			T::Staking::withdraw_unbonded(pool.bonded_account(), num_slashing_spans)?;
```

Comment:
So the removed `withdraw_from_staking` was a combination of calling `T::Staking::withdraw_unbonded` and updating `TotalValueLocked`, but this is now happening via `on_withdraw` after `T::Staking::withdraw_unbonded` is being called directly, thus rendering it unneeded now.

Reply:
Yes correct, that's the idea!